itenium-socks/src/app/socks/shop.component.ts

```diff
@@ -1,13 +1,13 @@
-import { Component } from '@angular/core';
+import {Component, LOCALE_ID} from '@angular/core';
```

Comment:
When joining a project, try to follow the coding standards/guidelines of the existing code!

Reply:
Also avoid adding code that is not in use! `LOCALE_ID` is not used here!